lua/earmuffs/server/sv_hl2_misc.lua

```diff
@@ -0,0 +1,25 @@
+local hl2CombatSounds = {
+    -- Misc
+    ["Flesh.BulletImpact"] = true,
+    ["Player.Death"] = true,
+    ["Player.FallDamage"] = true,
+    ["BaseCombatCharacter.AmmoPickup"] = true,
+    ["BaseCombatCharacter.StopWeaponSounds"] = true,
+}
+
+local utils = CFCEarmuffs.Utils
+
+local function isHL2Misc( soundData )
+    local maintainCase = true
+    local soundName = utils.CleanSoundName( soundData.OriginalSoundName, maintainCase )
+
+    if hl2CombatSounds[soundName] then return true end
```

Comment:
```suggestion
    return hl2CombatSounds[soundName]
```

Reply:
Going to leave it trailing in case we want more checks in there in the future - I don't want to insist that this be the last check